src/taoensso/sente/server_adapters/undertow.clj

```diff
@@ -1,8 +1,8 @@
 (ns taoensso.sente.server-adapters.undertow
-  "Sente server adapter for ring-undertow-adapter."
+  "Sente server adapter for ring-undertow-adapter.
+   Modified to avoid core.async and use promise directly and with read timeout"
```

Comment:
Please remove the docstring change here.
The mention of `core.async` seems like an implementation detail, and the read timeout we can better document elsewhere (e.g. in the constructor).